client/src/pages/Home/Home.js

```diff
@@ -30,22 +30,22 @@ class Home extends Component {
   render() {
     return (
       <>
-        <section class="landing">
-          <div class="landing-inner">
+        <section className="landing">
+          <div className="landing-inner">
             <Container>
-              <Row centered>
+              <Row>
```

Comment:
do we need the centered modifier?

Reply:
this was giving an error. so I removed it. it did not change the styling on the page so I'm not sure what the desired outcome was for this particular styling.